packages/react-native/src/assets/icons/index.ts

```diff
@@ -5,6 +5,8 @@ const icons = {
   checkboxOutline: require('./checkboxOutline.png') as ImageSourcePropType,
   close: require('./close.png') as ImageSourcePropType,
   error: require('./error.png') as ImageSourcePropType,
+  visibility: require('./visibility.png') as ImageSourcePropType,
```

Comment:
Nittish for sure but maybe rename to `visibilityOn` to create a clear relationship between it and the `visibilityOff` icon